tools/errortracker/errortracker.go

```diff
@@ -174,9 +175,15 @@ func handle(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	exception := r.URL.Query().Get("s")
+	// If format does not end with :\d+ truncate up to the last newline.
+	if !regexp.MustCompile(`:\d+\n?$`).MatchString(exception) {
```

Comment:
got it. i was testing in node earlier and the output had newline for whatever reason